.tekton/copyconfig.sh

```
@@ -12,40 +12,30 @@ if [ "$1" == "26" ]; then
 if [ -f "$NEW_PUSH_FILE" ]; then
     echo "$NEW_PUSH_FILE already exists"
     exit 1
 fi
 
-
-if [ -f "$NEW_PULL_FILE" ]; then
-    echo "$NEW_PULL_FILE already exists"
-    exit 1
-fi
-
 cp $BASE_PUSH_FILE $NEW_PUSH_FILE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)